components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -52,13 +52,15 @@ private const val VIEW_HOLDER_TYPE_ADDON = 2
  * @property addonsManagerDelegate Delegate that will provides method for handling the add-on items.
  * @param addons The list of add-on based on the AMO store.
  * @property style Indicates how items should look like.
+ * @property excludedAddons The list that should be excluded in UI in Recommended section
```

Comment:
nit: excludedAddons The list of of add-on IDs to be excluded from the recommended section.